uint32 seq		# sequence ID: consecutively increasing ID 
time stamp		# Two-integer timestamp of data
string agent_id		# Robot Data is associated with
float64 x		# X-coordinate of robot
float64 y		# Y-coordinate of robot
float64 yaw		# Yaw (heading) of robot
